app/_layout.tsx

```diff
@@ -33,10 +33,21 @@ import {
 import merge from 'deepmerge';
 import useColorScheme from '@/hooks/useColorScheme';
 import useCachedResources from '@/hooks/useCachedResources';
+import { isRunningInExpoGo } from 'expo';
+
+const routingInstrumentation = new Sentry.ReactNavigationInstrumentation();
 
 Sentry.init({
-  dsn: 'https://b852c07fe977471c96a3fb2dc1e10a49@o446803.ingest.sentry.io/4505356514426880',
-  debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
+  dsn: process.env.EXPO_SENTRY_DSN,
```

Comment:
Is dit nog steeds dezelfde URL?
\+ als in env, ook zorgen dat deze daar ook effectief in bestaat.

Reply:
Yes, staat in .env en in de verschillende environments (eas en gh actions) ingesteld